### PR TITLE
Tcs 20 add segmented subscriptions filter and only calculate symbol segments for new symbols when mapping

### PR DESCRIPTION
**Pull request by AdamBrown2**

### CODE REVIEW

File: tests/stp/subsegment/runstptests.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./torq.sh start discovery1
+./torq.sh start stp1
+taskset -c 0,1 /usr/bin/rlwrap q ${TORQHOME}/torq.q -proctype segmentedtickerplant -procname stp1 -test ./tests  -procfile ${KDBTESTS}/stp/subsegment/process.csv -schemafile ${TORQAPPHOME}/database.q -load ${KDBCODE}/processes/segmentedtickerplant.q -debug
+./torq.sh stop all
```
Comment (rian-morgan) [inline tests/stp/subsegment/runstptests.sh:6]: rename file to run.sh

Comment (rian-morgan) [reply tests/stp/subsegment/runstptests.sh:6]: relative path to torq.sh is probably going to break use ${TORQHOME}/torq.sh

**Review by rian-morgan — CHANGES_REQUESTED** [review]

**Review by rian-morgan — APPROVED** [review]

**Review by rian-morgan — APPROVED** [review]

File: tests/stp/subsegment/run.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+taskset -c 0,1 /usr/bin/rlwrap q ${TORQHOME}/torq.q -proctype segmentedtickerplant -procname stp1 -test ${KDBTESTS}/stp/subsegment/  -procfile ${KDBTESTS}/stp/subsegment/process.csv -schemafile ${TORQAPPHOME}/database.q -load ${KDBCODE}/processes/segmentedtickerplant.q -debug
```
Comment (rian-morgan) [inline tests/stp/subsegment/run.sh:3]: change this to QCMD plz

Comment (rian-morgan) [reply tests/stp/subsegment/run.sh:3]: plz format this in a more readable way, one arg per line

**Review by rian-morgan — CHANGES_REQUESTED** [review]

**Review by rian-morgan — APPROVED** [review]

**Review by jwrno1 — APPROVED** [review]

**Pull request merged**